docs/man-pages/condor_continue.rst

```diff
@@ -61,9 +76,20 @@ success, and it will exit with the value 1 (one) upon failure.
 Examples
 --------
 
+To continue a specific job:
+
+.. code-block:: console
+
+    $ condor_continue 432.1
+
 To continue all jobs except for a specific user:
 
 .. code-block:: console
 
     $ condor_continue -constraint 'Owner =!= "foo"'
```

Comment:
Can we change the prompt here from `$` to `#` to subtly indicate this command should be run as root?